src/sonic-py-common/setup.py

```diff
@@ -3,6 +3,7 @@
 dependencies = [
     'natsort==6.2.1', # 6.2.1 is the last version which supports Python 2
     'pyyaml',
+    'redis-dump-load',
```

Comment:
>redis-dump-load [](http://example.com/codeflow?start=5&length=15)

Thanks for the fixing!
Since we build redis-dump-load with patch, we never expect setup.py automatically download it from pypi. We enforce this rule by treating it as "sonic_dependencies". Could you follow the sample in [https://github.com/sonic-net/sonic-buildimage/pull/13850/files](https://github.com/sonic-net/sonic-buildimage/pull/13850/files)

Reply:
Fixed, also create a manually cherry-pick PR for 202211 and 202205: https://github.com/sonic-net/sonic-buildimage/pull/14360